wordpress-mobile/gutenberg-mobile
# Specify domain when extracting strings from Jetpack and Layout Grid

I noticed that we're adding the string `Cancel` in the localization strings files, which is referenced in the Layout Grid plugin ([file reference](https://github.com/Automattic/block-experiments/blob/00dbb963f1615d9843f0d8a8479b585c7a0108e1/blocks/layout-grid/src/grid/variation-control/index.native.js#L160-L162)). The string is retrieved from the default domain, which in this case it would be Gutenberg, and it's already translated in the GlotPress project ([reference](https://translate.wordpress.org/projects/wp-plugins/gutenberg/stable/en-gb/default/?filters%5Bterm%5D=Cancel&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc)), so we shouldn't really add it.

The `makepot` commands we use for extracting the strings from plugins like Jetpack and Layout Grid were ignoring the domain, which like the `Cancel` string case, they might include undesired extra strings into the localization strings files. This PR updates those commands and specify a domain for those plugins to prevent this behavior.

**NOTE:** A side effect of this change, although I guess it's expected, is that strings referencing a different domain than the one associated with the plugin (i.e. Jetpack references the` jetpack` domain) retrieved in `*.native.js` files of plugins, won't be included in the localization strings files.

**To test:**
1. Run command `npm run genstrings`.
2. Observe that the localization strings files (`bundle/ios/GutenbergNativeTranslations.swift` and `bundle/android/strings.xml`) remain unmodified.

PR submission checklist:
